### PR TITLE
Remove SemVer

### PR DESCRIPTION
**Pull request by mdedetrich**
Due to https://github.com/apache/pekko-persistence-r2dbc/issues/90

### CODE REVIEW

**Review by jxnu-liguobin — APPROVED** [review]

**Comment by mdedetrich** [comment]
@pjfanning Should I go ahead and merge this as its related to an issue you are talking about?

**Comment by pjfanning** [comment]
I'm neutral on this. The change proposed in #90 is not massively impacting but I don't mind if we change the semver setting.

**Comment by mdedetrich** [comment]
I would like to see https://github.com/apache/pekko-persistence-r2dbc/issues/90 done as it implied there is a legitimate reason why it was changed to `Long`. I'll merge this when I get home

**Comment by pjfanning** [comment]
@mdedetrich R2dbcExecutor is documented to be an internal class. For me, this means we can change its return types without too much fanfare.

**Comment by pjfanning** [comment]
Maybe we should work out how big the changes in https://github.com/apache/pekko-persistence-r2dbc/pull/101 will be before we decide if we need this.

**Comment by mdedetrich** [comment]
> Maybe we should work out how big the changes in https://github.com/apache/pekko-persistence-r2dbc/pull/101 will be before we decide if we need this.
> 
> So far the public API changes are only in pekko-projection-r2dbc and not in the pekko-persistence-r2dbc module - maybe we could just remove semver marker from pekko-projection-r2dbc.

Sure, do you want to go ahead with the PR to find out?

**Comment by gstein** [comment]
I removed myself as a reviewer. That was uncool @mdedetrich.

**Comment by mdedetrich** [comment]
> I removed myself as a reviewer. That was uncool @mdedetrich.

Apologies, this was an accident as there is another greg as part of committers.

**Pull request merged**
